content/zh-cn/docs/concepts/policy/resource-quotas.md

```diff
@@ -362,17 +362,17 @@ The following types are supported:
 支持以下类型：
 
 <!--
-| Resource Name | Description |
-| ----------------------------|--------------------------------------------- |
-| `configmaps` | The total number of ConfigMaps that can exist in the namespace. |
-| `persistentvolumeclaims` | The total number of [PersistentVolumeClaims](/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) that can exist in the namespace. |
-| `pods` | The total number of Pods in a non-terminal state that can exist in the namespace.  A pod is in a terminal state if `.status.phase in (Failed, Succeeded)` is true.  |
-| `replicationcontrollers` | The total number of ReplicationControllers that can exist in the namespace. |
-| `resourcequotas` | The total number of ResourceQuotas that can exist in the namespace. |
-| `services` | The total number of Services that can exist in the namespace. |
-| `services.loadbalancers` | The total number of Services of type `LoadBalancer` that can exist in the namespace. |
-| `services.nodeports` | The total number of Services of type `NodePort` that can exist in the namespace. |
-| `secrets` | The total number of Secrets that can exist in the namespace. |
+| Resource Name | Description                                                                                                                                                        |
+| ------------------------------- |--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `configmaps` | The total number of ConfigMaps that can exist in the namespace.                                                                                                    |
+| `persistentvolumeclaims` | The total number of [PersistentVolumeClaims](/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) that can exist in the namespace.                   |
+| `pods` | The total number of Pods in a non-terminal state that can exist in the namespace.  A pod is in a terminal state if `.status.phase in (Failed, Succeeded)` is true. |
+| `replicationcontrollers` | The total number of ReplicationControllers that can exist in the namespace.                                                                                        |
+| `resourcequotas` | The total number of ResourceQuotas that can exist in the namespace.                                                                                                |
+| `services` | The total number of Services that can exist in the namespace.                                                                                                      |
+| `services.loadbalancers` | The total number of Services of type `LoadBalancer` that can exist in the namespace.                                                                               |
```

Comment:
Seems these changes are problematic due to numerous redundancies.

Reply:
These changes are unnecessary.